server/src/main/java/org/elasticsearch/cluster/metadata/DataLifecycle.java

```diff
@@ -45,6 +46,9 @@ public class DataLifecycle implements SimpleDiffable<DataLifecycle>, ToXContentO
     private static final FeatureFlag DLM_FEATURE_FLAG = new FeatureFlag("dlm");
 
     public static final DataLifecycle EMPTY = new DataLifecycle();
```

Comment:
This is probably worth a comment, to contrast it with NULL? It might also be worth declaring it as
```
public static final DataLifecycle EMPTY = new DataLifecycle(null, false)
```
just so it's clear how it differs from NULL?

Reply:
Great suggestion!